paypal/standard/ipn/views.py

```diff
@@ -31,6 +35,11 @@ def ipn(request):
     flag = None
     ipn_obj = None
 
+    # Avoid the RawPostDataException. See original issue for details:
+    # https://github.com/spookylukey/django-paypal/issues/79
+    assert request.META.get('CONTENT_TYPE', '').startswith(
+        'application/x-www-form-urlencoded'), CONTENT_TYPE_ERROR
+
```

Comment:
You're right, I was just being "lazy". If I'm going to be touching this I don't mind updating the tests. In your opinion, would another exception, such as "TypeError", make more sense here?